{
  assertNoWarning("function f() { try { var x = 2; } catch (x) {} }");
  assertNoWarning("function f(e) { e = 3; try {} catch (e) {} }");
}
